CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone.
 # 10.10.0-rc.1
 ## Bug fixes 🐞
 * Trigger repaint after `BitmapWidget` is updated. ([1797](https://github.com/mapbox/mapbox-maps-android/pull/1797))
+* Fix bitmap widgets not rendering on specific devices. ([1834](https://github.com/mapbox/mapbox-maps-android/pull/1834))
```

Comment:
Should we mention that for `MapView` they need to change to `SHARED` context?

Reply:
revisited